tests/testdata/codegen/azure-native-nested-types/docs/documentdb/sqlresourcesqlcontainer/_index.md

```diff
@@ -347,7 +347,87 @@ Coming soon!
 
 
 
-## Create SqlResourceSqlContainer Resource {#create}
+## Creating a SqlResourceSqlContainer Resource
+<div>
+<pulumi-chooser type="language" options="typescript,python,go,csharp,java,yaml"></pulumi-chooser>
+</div>
+
+
+<div>
+    <pulumi-choosable type="language" values="csharp">
+        <div class="highlight">
+            <pre class="chroma">
+                <code class="language-csharp" data-lang="csharp">
+var sqlResourceSqlContainerResource = new AzureNative.DocumentDB.SqlResourceSqlContainer("sqlResourceSqlContainerResource");
+
+});
```

Comment:
This doesn't look right

Reply:
Nice catch! there was a bug with extracting resource examples that have no inputs. Fixed now and added tests ✅